tests/affine-migration/e2e/basic.spec.ts

```diff
@@ -80,81 +79,58 @@ test('v2 to v4, database migration', async ({ page }) => {
   await clickEdgelessModeButton(page);
   await page.waitForTimeout(200);
   expect(await page.locator('affine-database').isVisible()).toBe(true);
-
-  const changedLocalStorageData = await page.evaluate(() =>
-    window.readAffineLocalStorage()
-  );
-  const workspaces = JSON.parse(
-    changedLocalStorageData['jotai-workspaces']
-  ) as any[];
-  for (const workspace of workspaces) {
-    expect(workspace.version).toBe(4);
-  }
 });
 
 test('v3 to v4, surface migration', async ({ page }) => {
   const { localStorageData } = await open404PageToInitData(page, '0.8.4');
 
-  //#region fixme(himself65): blocksuite issue, data cannot be loaded to store
-  const allPagePath = `${coreUrl}/workspace/${localStorageData.last_workspace_id}/all`;
-  await page.goto(allPagePath);
-  await page.waitForTimeout(5000);
-  //#endregion
-
   const detailPagePath = `${coreUrl}/workspace/${localStorageData.last_workspace_id}/${localStorageData.last_page_id}`;
   await page.goto(detailPagePath);
+  await page.waitForTimeout(3000);
```

Comment:
playwright doesn't recommend using waitForTimeout, which can cause erratic results.

https://playwright.dev/docs/api/class-frame#frame-wait-for-timeout